backend/canisters/group_index/impl/src/updates/set_max_concurrent_group_canister_upgrades.rs

```diff
@@ -0,0 +1,27 @@
+use crate::guards::caller_is_controller;
+use crate::read_state;
+use canister_tracing_macros::trace;
+use group_index_canister::set_max_concurrent_group_canister_upgrades::{Response::*, *};
+use ic_cdk_macros::update;
+use tracing::info;
+
+// dfx --identity openchat canister --network ic call group_index set_max_concurrent_canister_upgrades '(record { value=N:nat64 })'
+#[update(guard = "caller_is_controller")]
+#[trace]
+async fn set_max_concurrent_group_canister_upgrades(args: Args) -> Response {
+    let canisters: Vec<_> = read_state(|state| state.data.local_index_map.canisters().copied().collect());
+
+    let args = local_group_index_canister::c2c_set_max_concurrent_group_upgrades::Args {
+        value: args.value as u32,
+    };
+
+    let futures: Vec<_> = canisters
+        .into_iter()
+        .map(|canister_id| local_group_index_canister_c2c_client::c2c_set_max_concurrent_group_upgrades(canister_id, &args))
+        .collect();
+
+    futures::future::join_all(futures).await;
```

Comment:
We should check that they all actually succeeded.

That could be a follow on PR though, but we should label it as a TODO.